lib/trackler/implementation.rb

```diff
@@ -91,12 +89,19 @@ def readme_title
       problem.name
     end
 
+    def optional_blurb
+      blurb = problem.blurb
+      return '' if problem.description.start_with?(blurb)
+      "#{blurb}\n\n"
+    end
```

Comment:
The complete plan is documented here: https://github.com/exercism/x-common/issues/767